bundles/net.tourbook.chart/src/net/tourbook/chart/ChartDrawingData.java

```diff
@@ -27,7 +27,7 @@ public class ChartDrawingData {
    ChartDataModel                      chartDataModel;
 
    int                                 devMarginTop;
-   int                                 devXTitelBarHeight;
+   int                                 devXTitleBarHeight;
    int                                 devSliderBarHeight;
```

Comment:
This issue exists since the beginning

![image](https://github.com/user-attachments/assets/22d59938-4881-4e80-bfe8-b6c7393ec240)